website/docs/language/import/index.mdx

```diff
@@ -131,3 +132,32 @@ import {
   id = "i-abcd1234"
 }
 ```
+
+### Importing multiple resources
+
+You can import multiple resources with one import block by using a `for_each` expression. This expression accepts a [set or a map](/docs/language/expressions/types/) and provides the `each.key` and `each.value` variables to access the individual elements.
```

Comment:
We also support `tuple` in the case of for_each. Using a tuple can be helpful when importing to a more complex address like `module.mod[each.value.moduleKey].test_object.a[each.value.resourceKey]`.